api/handlers/container/stop.go

```diff
@@ -24,8 +26,30 @@ func (h *handler) stop(w http.ResponseWriter, r *http.Request) {
 	}
 	timeout := time.Second * time.Duration(t)
 
+	signal := r.URL.Query().Get("signal")
+	if signal == "" {
+		signal = "SIGTERM" // Docker/nerdctl default
```

Comment:
Nit: Should we just let nerdctl use the default then, if we are not opinionated here?

Reply:
removed the explicit SIGTERM setting to let nerdctl handle default behavior